app/Run.hs

```diff
@@ -160,7 +155,7 @@ runNode nodeCli@NodeCLIArguments{..} loggingLayer cc = do
 -- | Sets up a simple node, which will run the chain sync protocol and block
 -- fetch protocol, and, if core, will also look at the mempool when trying to
 -- create a new block.
-handleSimpleNode :: forall blk. (RunNode blk, TraceConstraints blk)
+handleSimpleNode :: forall blk. (RunNode blk, TraceConstraints blk, Show (GenTxId blk))
```

Comment:
This constraint is already part of `TraceConstraints`, see https://github.com/input-output-hk/cardano-node/blob/a380dab410031020cded1caf1798aaa155e5520d/cardano-node/src/Cardano/Node/CLI.hs#L108